vcf-to-gds-wf.wdl

```diff
@@ -155,10 +155,7 @@ task check_gds {
 	input {
 		File gds
 		Array[File] vcfs
-		String gzvcf = basename(sub(gds, "\\.gds$", ".vcf.gz"))
-		String bgzvcf =  basename(sub(gds, "\\.gds$", ".vcf.bgz"))
-		String uncompressed = basename(sub(gds, "\\.gds$", ".vcf"))
-		String bcf = basename(sub(gds, "\\.gds$", ".bcf"))
+		File? sample
```

Comment:
A better name would be sample_list or sample_file, etc, but this isn't a big deal.

Reply:
Leaving as is to keep one word variable consistency